app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt

```diff
@@ -102,7 +114,27 @@ class ProfileFragment : Fragment(), ProfileView, ActionMode.Callback {
         }
     }
 
+    fun showWidechatProfile(avatarUrl: String, name: String, username: String, email: String?) {
+        ui {
+            image_avatar.setImageURI(avatarUrl)
+            widechat_text_name.text = name
+            widechat_text_username.textContent = username
+            widechat_text_email.textContent = email ?: ""
+
+            currentName = name
+            currentUsername = username
+            currentEmail = email ?: ""
+
```

Comment:
Are the above "currentXxxx"'s actually only for when the fields are editable and therefore N/A?

Reply:
Good point.  I'll remove those.